homework/src/index.js

```diff
@@ -33,14 +33,123 @@
   function main(url) {
     fetchJSON(url, (err, data) => {
       const root = document.getElementById('root');
+      const header = createAndAppend('header', root);
+      createAndAppend('p', header, {html: 'HYF Repositories', class: 'header-lable'});
+      const select = createAndAppend('select', header, { id: 'selectMenu' });
       if (err) {
         createAndAppend('div', root, { html: err.message, class: 'alert-error' });
       } else {
-        createAndAppend('pre', root, { html: JSON.stringify(data, null, 2) });
+        //createAndAppend('pre', root, { html: JSON.stringify(data, null, 2) });
+        /* &&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&& */
+        data.sort((a, b) => a.name.localeCompare(b.name, 'fr', {ignorePunctuation: true}));
+        data.forEach((reopsitory, index) => {
+          createAndAppend('option', select, {html: reopsitory.name, value: index});
+        });
+        /* &&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&& */
+        const container = createAndAppend('div', root, { id: 'container'});
+        contentLeft(container, data, select);
+        contentRight(container, data, select);
       }
     });
   }
 
+
+
+
+
+  // function who creates a left content section
+  function contentLeft(container, data, select) {
+    const leftSection = createAndAppend('div', container, { id: 'leftSection', class: 'whiteframe' });
+    showTableRepo(leftSection, data, 0);
```

Comment:
I would not pass the who repository array + index to the `showTableRepo` function. This latter function does not need the whole array, it just needs the selected repo. So you could pass it as:

```js
showTableRepo(leftSection, data[0]);
```

Reply:
Oh! 
It's spelling error and i wanted to passed just the element data[0]

Reply:
fixed